pkg/trace/stats/sublayers.go

```diff
@@ -46,48 +129,39 @@ func (v SublayerValue) GoString() string {
 //       <-5------------------->
 //                         <--6-------------------->
 //                                             <-7------------->
-// 1: service=web-server, type=web,   parent=nil
-// 2: service=pg,         type=db,    parent=1
-// 3: service=render,     type=web,   parent=1
-// 4: service=pg-read,    type=db,    parent=2
-// 5: service=redis,      type=cache, parent=1
-// 6: service=rpc1,       type=rpc,   parent=1
-// 7: service=alert,      type=rpc,   parent=6
+// idx 0 id 1: service=web-server, type=web,   parent=nil
+// idx 1 id 2: service=pg,         type=db,    parent=1
+// idx 2 3: service=render,     type=web,   parent=1
+// idx 3 4: service=pg-read,    type=db,    parent=2
+// idx 4 5: service=redis,      type=cache, parent=1
+// idx 5 6: service=rpc1,       type=rpc,   parent=1
+// idx 6 7: service=alert,      type=rpc,   parent=6
 //
 // Step 1: Find all time intervals to consider (set of start/end time
-//         of spans):
+//         of spans). For each timestamp, store the spanIdx, parentIdx and if it's the open or close timestamp:
+// if the parent span is not in the trace, parent idx is set to len(trace)
 //
-//         [0, 10, 15, 20, 50, 60, 70, 80, 110, 120, 130, 150]
+//         [(0, open, span idx 0, parent idx 7), (10, open, span idx 1, parent idx 0), ... (150, close, span idx 6, parent idx 5)]
```

Comment:
At this point, where we have the word `open` and `close` in the documentation, these two concepts should be crystal clear. What is `open`? What is `close`? Please explain it well in the documentation. The current documentation makes it very easy to understand the algorithm. We should keep it like that.